### ISSUE TITLE
Select components do not respect tab index

### ISSUE THREAD

**Issue by ajcolub11** [formio/formio.js#186]
It seems as though select components are not respecting the tab index values set in form.io portal.

Here is how our tab indexes are set up (marked in red):
![image](https://user-images.githubusercontent.com/29924975/30647761-4b3f8916-9dea-11e7-9d09-8ead6cbb1175.png)

Behavior on form.io preview:
When viewing the form above on form.io preview and tabbing through the fields, once the user tabs into field #8 (Marital Status), then the tab index proceeds to prioritize the other select components below, so the order of tabbing is as follows: 1, 2, 3, 4, 5, 6, 7, 8, 10, 12, 13, 14
**However, when values are set for these select components, the tab index is working as expected.**

Behavior on vanilla js renderer:
Select components are being ignored by tab index. So, the order of tabbing on our front-end is as follows: 1, 2, 3, 4, 5, 6, 7, 9, 11, 13, 14
Setting the values for these select components does not change this ordering.

Expected behavior:
Select components should respect the tab indexes set in form.io portal

**Comment by travist** [comment]
I am pretty sure that this is an issue with the Choices.js library which may make this one more difficult to track down.

**Comment by vchtelmakh** [comment]
@travist Just wanted to get an update on this - is this in your backlog? If so, is there a possibility that it will be addressed in the near future?

**Comment by Aiso786** [comment]
Yes, the issue is with the Choices.js library. When you use Tab key to get focus on the Select component, it's the search field inside the Select that has the focus.

![image](https://user-images.githubusercontent.com/22521614/33939975-7c88fe16-e00d-11e7-9346-dff1b2a8743a.png)
